static/skin/taskbar.js

```diff
@@ -5,18 +5,26 @@ function htmlDecode(input) {
 
 (function ($) {
     const root = $( `link[type='root']` ).attr("href");
+    const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
 
     $( "#kiwixsearchbox" ).autocomplete({
 
-        source: `${root}/suggest?content=${window.location.pathname.split(`${root}/`)[1].split('/')[0]}`,
+        source: `${root}/suggest?content=${bookName}`,
         dataType: "json",
         cache: false,
 
         response: function( event, ui ) {
 
               for(const item of ui.content) {
                   item.label = htmlDecode(item.label);
-                  item.value = htmlDecode(item.value);
+                  item.value = encodeURI(htmlDecode(item.value));
+
+                  if (item.label === `containing '${item.value}'...`) {
```

Comment:
Even if this condition holds true for the original values of `item.label` and `item.value`, it may fail to be so for their transformed values.